Sources/Hyperspace/HTTP/HTTP.swift

```diff
@@ -127,53 +127,93 @@ public struct HTTP {
             self.init(data)
         }
     }
+
+    /// Represents an HTTP request
+    public struct Request: Equatable {
+
+        /// The `URL` to which the `Request` originated.
+        public let url: URL?
+
+        /// The HTTP header fields for this request.
+        public let headers: [String: String]?
+
+        /// The raw `Data` associated with the HTTP request, if any was provided.
+        public let body: Data?
+
+        /// Initialize a new `Request` with any given `URL`, HTTP headers and body.
+        /// - Parameters:
+        ///   - url: The `URL` for which this request was created.
+        ///   - data: The raw `Data` associated with the HTTP request, if any was provided.
+        ///   - headers: The HTTP header fields for this request.
```

Comment:
The parameter comments here are out of order and don't match up to the method signature below.

Reply:
Fixed